exercises/dot-dsl/.meta/description.md

```diff
@@ -0,0 +1,22 @@
+Write a Domain Specific Language similar to the Graphviz dot language.
```

Comment:
For some reason `configlet generate`  did not generate README unless i manually copied the description from `problem-specification`

Reply:
That's interesting. Could it be that `configlet generate` is referencing a local copy of the `problem-specifications` repo which has fallen out of date?

Reply:
I actually updated the description of `dot-dsl` on the `problem-specification` repo just recently, so no, it is up to date

Reply:
I could not replicate this behaviour - `generate` does cause `dot-dsl/README.md` to be written, even without this file. I would prefer not to have this file because we don't want to have to maintain a duplicate version of it.